src/Stratis.Bitcoin.Features.SmartContracts/Models/LocalCallContractRequest.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Text;
+using Stratis.Bitcoin.Features.SmartContracts.ReflectionExecutor.Consensus.Rules;
+using Stratis.Bitcoin.Features.Wallet.Validations;
+
+namespace Stratis.Bitcoin.Features.SmartContracts.Models
+{
+    public class LocalCallContractRequest
+    {
+        [Required(ErrorMessage = "A destination address is required.")]
+        [IsBitcoinAddress]
+        public string ContractAddress { get; set; }
+
+        [Required(ErrorMessage = "A method name is required.")]
+        public string MethodName { get; set; }
+
+        [Required(ErrorMessage = "An amount is required.")]
+        public string Amount { get; set; }
```

Comment:
Also think this doesn't need to be required. Most common use of `local-call` will be to call something like `TotalSupply`. What do you think?

Reply:
I think it's necessary. Consider times you might want to do a calculation with the amount transferred and return that as a result.

Reply:
Removed, though GitHub is not showing it as outdated here.